### DISCUSSION TITLE
In-game Tutorial Mode [Feature Suggestion]

### DISCUSSION THREAD

**Discussion by Zagamun** [unitystation/unitystation#8418] [Ideas]
## Description
I think it would be a good idea to have an in-game tutorial mode for teaching roles to people while excluding explanations for content that isn't in the game yet.

When launching the game like you would when hosting a local server, there could be an option for the tutorials. This would then lead to a dialogue where you can choose a role, like the one currently in the game. You would then spawn into a locally hosted server with a map designed specifically for teaching you a certain role or department. 

For example, if you chose to learn a cargo role you would spawn in a map that only has a cargo department and has various signs, books, or paper documents lying around that teach you all the cargo roles.

Something like an interactive tutorial with dialogue boxes popping up and showing you everything step by step would be much less easily editable and would require a role to have all their features coded in for a proper teaching flow. With the method of just having items that describe everything, you would be able to easily add elements to the tutorial as features are added to the game. Thus allowing for having WIP tutorials while the game is still adding in the features for each role. 

## TL;DR
Locally hosted tutorial maps dedicated to a specific department with in-game objects that teach you how to play those roles. If having multiple different tiny maps (not made for actual gameplay purposes) would be inefficient then possibly just one big tutorial map with instruction items for every role.
